fastlane/Appfile

```diff
@@ -0,0 +1,4 @@
+app_identifier("com.itcraftship.babyNames") # The bundle identifier of your app
+apple_id("kuba@danielczyk.pl") # Your Apple email address
```

Comment:
I think we should remove this:
1. It makes your email public ;)
2. Each collaborator should be able to log in with their own Apple ID. If we don't set it here, Fastlane will ask about it every time which is desired. For deploying to AppStore, we should use the app-specific password anyway.